src/component/facebook/FaceBook.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react'
+import profiles from '../../data/berlin.json';
+import './FaceBook.css'
+
+class FaceBook extends Component {
```

Comment:
Really, really nice! You got all fancy with the pic toogles and stuff. Although, to make it perfect you just need to style it a bit.